### ISSUE TITLE
Show a big-fat warning when a brain mask is not provided

### ISSUE THREAD

**Issue by oesteban** [nipreps/eddymotion#107]
With the development of #91, it has become clear that we don't want to be fitting models in background pixels, nor have the background be considered by the cost function of ANTs.

Therefore, eddymotion should be very reluctant to go ahead without a brain mask.

Incidentally, I believe can easily include SynthStrip into dMRIPrep and then dilate that mask so that distortions do not cause cropping of the brain.

WDYT?

**Comment by arokem** [comment]
I ❤️ synthstrip and I think that's a great idea. We should also give the option to provide masks from other pipelines.

**Comment by oesteban** [comment]
Correct, we don't want to mess with how the user calculates masks - eddymotion should just trust the user is doing the right thing on this aspect.
